packages/components/src/VirtualizedList/CellBoolean/CellBoolean.test.js

```diff
@@ -1,38 +1,31 @@
 import React from 'react';
-import { shallow } from 'enzyme';
+import { mount } from 'enzyme';
 
 import CellBoolean, { DISPLAY_MODE } from './CellBoolean.component';
 
 describe('CellBoolean', () => {
 	it('should render an empty cell', () => {
-		const wrapper = shallow(<CellBoolean />);
-
-		expect(wrapper.getElement()).toMatchSnapshot();
+		const wrapper = mount(<CellBoolean />);
+		expect(wrapper.html()).toBe(null);
 	});
 
-	it('should render an icon cell in a truthy case', () => {
-		const wrapper = shallow(
-			<CellBoolean cellData columnData={{ displayMode: DISPLAY_MODE.ICON }} />,
-		);
-
-		expect(wrapper.getElement()).toMatchSnapshot();
+	it('should render an check icon for a truthy value in icon mode', () => {
+		const wrapper = mount(<CellBoolean cellData columnData={{ displayMode: DISPLAY_MODE.ICON }} />);
+		expect(wrapper.html()).toMatchSnapshot();
```

Comment:
Print the html instead of a big overcomplicated wrapper to check that it's a div with an icon in it